tests/test_release_gates_do_not_block_good_documents.py

```
@@ -0,0 +1,187 @@
+    assert record.citable_verbatim, "норма прочитана с официального источника — её можно цитировать"
+    assert "adilet.zan.kz" in record.source_url
+
+
+def test_missing_article_is_still_unverifiable(corpus_with_620):
+    """Мост не должен подтверждать то, чего в корпусе нет."""
+    from korgan.provision_corpus import lookup
+
+    assert lookup("ГК РК", "99999") is None
+
+
+def test_citation_audit_accepts_a_corpus_backed_paraphrase(corpus_with_620):
```

Ruff (0.16.2)

[warning] Unused function argument: `corpus_with_620`

(ARG001)